rclcpp/include/rclcpp/experimental/buffers/intra_process_buffer.hpp

```diff
@@ -94,6 +94,8 @@ class TypedIntraProcessBuffer : public IntraProcessBuffer<MessageT, Alloc, Messa
 
     buffer_ = std::move(buffer_impl);
 
+    TRACEPOINT(buffer_to_typedIPB, buffer_.get(), static_cast<const void *>(this));
```

Comment:
As per my suggestions in the `ros2_tracing` PR. Also, there's a missing `static_cast`.

```suggestion
    TRACEPOINT(
      buffer_to_ipb,
      static_cast<const void *>(buffer_.get()),
      static_cast<const void *>(this));
```

Reply:
Also, you should `#include "tracetools/tracetools.h"` in this file.

Reply:
Thank you for your review @christophebedard.
fixed in [9ead575](https://github.com/ros2/rclcpp/pull/2091/commits/9ead5753751ed694c7acac47483c6f37ef61919d).